crates/bitwarden/src/crypto/rsa.rs

```diff
@@ -0,0 +1,31 @@
+use base64::Engine;
+use rsa::{
+    pkcs8::{EncodePrivateKey, EncodePublicKey},
+    RsaPrivateKey, RsaPublicKey,
+};
+
+use crate::{
+    crypto::{encrypt_aes256_hmac, EncString, SymmetricCryptoKey},
+    error::{Error, Result},
+    util::BASE64_ENGINE,
+};
+
+pub(super) fn make_key_pair(key: &SymmetricCryptoKey) -> Result<(String, EncString)> {
```

Comment:
I feel like returning a tuple with two similar values like String and EncString could lead to confusion about which one is which. What do you think about creating a KeyPair struct with `private_key` and `public_key` fields?